server/server.py

```diff
@@ -5,6 +5,10 @@
 import json_config
 import pafy
 import vlc
+import sys
+import json
+sys.path.append(".")
+from youtube_videos import youtube_search
```

Comment:
Instead of using 'sys.path.append', use relative imports like 'from ./youtube_videos' import youtube_search

Reply:
yeah sorry, that was unnecessary, I've made the changes.

Reply:
@abhiroopdutta , it's still showing the same commit for me 😕